test/reducersTest.js

```diff
@@ -92,4 +93,32 @@ describe('reducers', function () {
       assert.equal(returnedState.broker.getAssetById(2).numOwned, 1);
     });
   });
+
+  describe('Action: UPGRADE_DENOMINATION', function () {
+    it('adds a news article', function () {
+      const returnedState = trumpMM(initialState, createAction.upgradeDenomination());
+      assert.equal(returnedState.news.articles.length, initialState.news.articles.length + 1);
+    });
+    describe('if there is insufficient cash', function () {
+      it('leaves original mint and bank in place', function () {
+        initialState.bank._cash = 0;
+        const returnedState = trumpMM(initialState, createAction.upgradeDenomination());
+        assert.equal(returnedState.bank, initialState.bank);
+        assert.equal(returnedState.mint, initialState.mint);
```

Comment:
since they're immutable objects, testing reference equality is all that's needed (and the exact benefit of using immutable objects).  reference equality => deep equality.  however, deep equality !=> reference equality.  here we're just using the first implication so we good